core/src/test/java/nl/nn/adapterframework/xslt/Xslt3Test.java

```diff
@@ -0,0 +1,123 @@
+package nl.nn.adapterframework.xslt;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
```

Comment:
Shall we move these tests to JUnit5 before merging or do you want to just get it merged?

Reply:
I contemplated that, I thought we might need to backport this to the 7.8 and thus it's better to leave it at JUnit 4 for now..